src/components/Inputs/input.module.scss

```diff
@@ -1438,6 +1447,132 @@
     &.input-stretch {
         width: 100%;
     }
+
+    &.isExpandable {
+        .expandable-wrapper {
+            position: relative;
+        }
+
+        .input-group {
```

Comment:
inputs are highly configurable. You'll need to test your animation against a few cases:

- Search icon button only
- Icon + Search Icon Button
- No Icon or Icon button
- Align Icon (left and right)
- RTL canvas dir when align icon is left

Reply:
Additionally, you'll need to test underline and rectangle shaped input all the above.

Reply:
this is done